radio/src/Makefile

```diff
@@ -850,6 +856,8 @@ ifeq ($(PCB), $(filter $(PCB), SKY9X 9XRPRO AR9X))
   ifeq ($(RTCLOCK), YES)
     CPPDEFS += -DRTCLOCK
     CPPSRC += rtc.cpp targets/sky9x/rtc_driver.cpp
+  else ifeq ($(PCB), SKY9X)
+    CPPSRC += targets/sky9x/rtc_driver.cpp
```

Comment:
Oh. When I mentioned `coproc_driver.cpp` I meant this as a suggestion for renaming `rtc_driver.cpp`, I wasn't aware that this file already exists. ;-)

Happy to do some cleaning up, and yes definitely *not* as part of this patch. I would think the cleanup will probably best be done against `next` anyway? (I really first need to understand more about the release process for this project.)